chainer/function.py

```diff
@@ -33,13 +48,23 @@ def no_backprop_mode():
     >>> x.grad is None
     True
 
+    .. note::
+
+       ``chainer.no_backprop_mode()`` implicitly applies ChainerX's
+       counterpart :func:`chainerx.no_backprop_mode()`, but not vice versa.
+       Also, setting ``enable_backprop`` :ref:`configuration <configuration>`
+       does not affect ChainerX.
+
     .. seealso::
 
-       See :func:`force_backprop_mode` for details on how to override this
-       context.
+       See :func:`chainer.force_backprop_mode` for details on how to override
+       this context.
 
     """
-    return configuration.using_config('enable_backprop', False)
+    c = configuration.using_config('enable_backprop', False)
+    if chainerx.is_available():
+        return _BackpropModeContext((c, chainerx.no_backprop_mode()))
+    return c
```

Comment:
How about returning `_BackpropModeContext((c,))` for consistency?

Reply:
It would be unncecessary overhead.